Dockerfile.sites-partners

```diff
@@ -0,0 +1,64 @@
+FROM node:12.16-alpine AS development
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+
+WORKDIR /usr/src/app/
+
+COPY . ./
```

Comment:
Can you add a comment about what's happening here.

Reply:
Done!